synopsys-task/task.json

```diff
@@ -163,7 +163,15 @@
       "label": "Add Comments to Pull Requests",
       "defaultValue": false,
       "required": false,
-      "helpMarkDown": "Applicable to pull requests only, Requires Azure Token.",
+      "helpMarkDown": "Applicable to pull requests only, Requires Azure Token."
+    },
+    {
+      "name": "bridge_polaris_test_sca_type",
```

Comment:
can you make it in camel case as we are following this pattern for all new fields. Accept this input as well along with bridge_polaris_test_sca_type